### ISSUE TITLE
Cityscapes instance

### ISSUE THREAD

**Issue by zhengduoru** [Wizaron/instance-segmentation-pytorch#5]
hello, me again. Execuse me.
I read the paper _Semantic Instance Segmentation with a Discriminative Loss _Function_
It is amazing that your model can be used with cityscape dataset.
I wanna know what should I do to apply your model on cityscape dataset.

**Comment by Wizaron** [comment]
Hey.

Cityscapes dataset has multiple classes, whereas CVPPP has only two classes (foreground (leaf) and background). Hence, this code currently assumes that there are two classes.

arch.py, model.py and prediction.py should be edited in order to allow multiple classes.

I am going to make an update to support multiple classes soon.

**Comment by zhengduoru** [comment]
Looking forward to multiple classes~
Thank you.

**Comment by Wizaron** [comment]
I have just created a branch named "cityscapes". You can find codes to train on cityscapes dataset in there. However, network architecture (number of filters etc.) needs to be updated according to the dataset.

I will merge it to master, after some tests and hyper-parameter tuning.

**Comment by pvthuy** [comment]
Sorry, any updates? @Wizaron 
I have tried with the branch "cityscapes" for multi-class instance segemntation, but I can't train the model using your code.
